persistence/src/commonTest/kotlin/com/wire/kalium/persistence/dao/MessageDAOBenchmarkTest.kt

```diff
@@ -0,0 +1,101 @@
+package com.wire.kalium.persistence.dao
+
+import com.wire.kalium.persistence.BaseDatabaseTest
+import com.wire.kalium.persistence.dao.message.MessageDAO
+import com.wire.kalium.persistence.dao.message.MessageEntity
+import com.wire.kalium.persistence.dao.message.MessageEntityContent
+import com.wire.kalium.persistence.utils.stubs.newConversationEntity
+import com.wire.kalium.persistence.utils.stubs.newUserEntity
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runTest
+import kotlin.random.Random
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.time.ExperimentalTime
+import kotlin.time.measureTime
+
+class MessageDAOBenchmarkTest : BaseDatabaseTest() {
```

Comment:
Can we `@Ignore` this one?

If we start adding Benchmarks, CI will be super slow.

This one takes some seconds to run which slows down CI, and it's not something we will be monitoring anyway.